src/components/modal/modal.st.css

```diff
@@ -9,7 +9,7 @@
     background: rgba(0, 0, 0, 0.05);
 }
 
-.children {
+.body {
```

Comment:
I don't know if this might cause any problem or confusion but naming a class `body` seems weird to me